.github/CODEOWNERS

```diff
@@ -747,10 +747,8 @@ docs/setup/running_docker.md    @department-of-veterans-affairs/backend-review-g
 docs/setup/running_natively.md    @department-of-veterans-affairs/backend-review-group
 docs/setup/va_forms.md    @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 docs/setup/virtual_machine_access.md    @department-of-veterans-affairs/backend-review-group
-# Gemfile
```

Comment:
I had left the comments in an attempt to follow their pattern of separating out sections with comments but 🤷